sample/Sample.Tizen/Sample.Tizen.csproj

```diff
@@ -17,10 +17,5 @@
     <ProjectReference Include="..\Sample\Sample.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'tizen40' ">
-    <TizenTpkUserIncludeFiles Include="$(SolutionDir)theme\*.edj">
-      <TizenTpkSubDir>res</TizenTpkSubDir>
-    </TizenTpkUserIncludeFiles>
-  </ItemGroup>
```

Comment:
This ItemGroup is for loading theme file properly when using project reference just like this sample project.
Is there any reason to remove it?

Reply:
done